scripts/meta/gatherer.py

```diff
@@ -0,0 +1,42 @@
+import json
+from website.settings import GITHUB_API_TOKEN
+
+the_filename = 'website/static/git_logs.json'
```

Comment:
Minor: Uppercase constants. Also, no need for articles; how about `GIT_LOGS_FILE`. Also, best to use `os.path.join` for building filepaths.